### ISSUE TITLE
Feature Request: Option to use tabs instead of spaces when aligning property value pairs

### ISSUE THREAD

**Issue by kborowinski** [PowerShell/vscode-powershell#945]
Is it possible to add option to align property value pairs with tabs instead of spaces?

```javascript
  // Align assignment statements in a hashtable or a DSC Configuration.
  "powershell.codeFormatting.alignPropertyValuePairs": true,
```

Currently when formatting hashtable the tabs are replaced with spaces:

![image](https://user-images.githubusercontent.com/16168755/28096570-1f6fd164-66aa-11e7-9710-508a3872b4cb.png)

Would love to have the option to use tabs for alignment instead:

![image](https://user-images.githubusercontent.com/16168755/28096534-db6a0eee-66a9-11e7-9caa-960766c9fb21.png)

**Comment by rkeithhill** [comment]
I don't think this needs a new PowerShell setting.  The code should honor the current `editor.insertSpaces` and `editor.tabSize` settings

**Comment by kborowinski** [comment]
I agree, but then it looks like the editor does not honor the `editor.insertSpaces` setting when aligning property value pairs. I've set `"editor.insertSpaces": false` and still the property value pairs are aligned with spaces.
